src/apis/staked-relayer.ts

```diff
@@ -24,6 +24,12 @@ export interface StakedRelayerAPI {
     getAllActiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllInactiveStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
     getAllStatusUpdates(): Promise<Array<{ id: u256; statusUpdate: StatusUpdate }>>;
+    getFees(stakedRelayerId: AccountId): Promise<PolkaBTC>;
+    getSLA(stakedRelayerId: AccountId): Promise<number>;
+    getScore(stakedRelayerId: AccountId): Promise<string>;
+    getTotalScore(): Promise<string>;
+    getTotalReward(): Promise<string>;
+    getReward(stakedRelayerId: AccountId): Promise<string>;
```

Comment:
Not sure how this is different from the getFees? IMO probably don't need that.

Reply:
See the screenshot attached